python/llm/example/CPU/PyTorch-Models/Model/mixtral/README.md

```diff
@@ -12,18 +12,34 @@ In the example [generate.py](./generate.py), we show a basic use case for a Mixt
 We suggest using conda to manage the Python environment. For more information about conda installation, please refer to [here](https://docs.conda.io/en/latest/miniconda.html#).
 
 After installing conda, create a Python environment for IPEX-LLM:
+
+On Linux:
+
 ```bash
 conda create -n llm python=3.11 # recommend to use Python 3.11
 conda activate llm
 
 # below command will install PyTorch CPU as default
 pip install torch==2.0.1 --index-url https://download.pytorch.org/whl/cpu
-pip install --pre --upgrade ipex-llm[all]
+pip install --pre --upgrade ipex-llm[all] --extra-index-url https://download.pytorch.org/whl/cpu
+
```

Comment:
Looks strange for me. I am not sure whether `torch` with version `2.0.1` is required. If not, we could remove the `pip install torch` part as it has been included into our `ipex-llm[all]`

Reply:
@qiyuangong What do you think :)?